### PR TITLE
Archived

### PR DESCRIPTION
**Pull request by LouMM**

### CODE REVIEW

**Comment by lindalu-MSFT** [comment]
@LouMM Give me a moment and I’ll fix the merge conflict.

**Comment by akrantz** [comment]
@LouMM I think it would be better to redo this and start with the latest master branch first. It looks like there are a bunch of changes in here which may be unintended.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-[![dependencies](https://david-dm.org/officedev/office-js-helpers.svg)](https://david-dm.org/officedev/office-js-helpers)
-[![downloads](https://img.shields.io/npm/dt/@microsoft/office-js-helpers.svg)](https://www.npmjs.com/package/@microsoft/office-js-helpers)
+> # ❗️ Deprecation Notice ❗️ [![No Maintenance Intended](badge.svg)](https://www.npmjs.com/package/@microsoft/office-js-helpers)
+>**Thank you** for all your efforts and contributions!
+> ``` office-js-helpers ``` is a community-driven package of sample code that encapsulates a set of convenience functions. The package was initially developed as an example abstraction to common patterns found by developers.
+> The community has been great at submitting ideas, issues, and fixes. As part of our latest campaign to ensure developers can quickly discover content and code, we are consolidating the many locations we post patterns and practices. To ensure we can focus our attention on the quality of content, we will be deprecating the ```office-js-helpers``` package and archiving this repository.
```
Comment (davidchesnut) [inline README.md]:
```suggestion
> The community has been great at submitting ideas, issues, and fixes. As part of our latest campaign to ensure developers can quickly discover content and code, we are consolidating the many locations we post patterns and practices. To ensure we can focus our attention on the quality of content, we have deprecated the ```office-js-helpers``` package and archived this repository.

```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
+> ## Can I still use this code?
+> If you find these samples, patterns, and convenience functions useful, we encourage you to take the code under your wing, improve them, and use it on your own. Even though Microsoft will no longer be supporting this content going forward, you may use the code at your convenience, external to the package.
```
Comment (davidchesnut) [inline README.md]:
```suggestion
> **Security vulnerabilities may exist in the project, or its dependencies.** If you plan to reuse or run any code from this repo, be sure to perform appropriate security checks on the code or dependencies first.
> If you find these samples, patterns, and convenience functions useful, we encourage you to take the code under your wing, fix any security issues, improve them, and use them on your own. Even though Microsoft will no longer be supporting this content going forward, you may use the code at your convenience, external to the package.
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
+> ## What will happen next?
+> This repository will be 📕**_Archived_**📕 and all issues and PR's will be archived along with it.
```
Comment (davidchesnut) [inline README.md]:
```suggestion

```
Redundant, just remove this line.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
+> 
+> To ensure that the community has some content and solutions for the common patterns and practices for Office Add-in, we have consolidated the content and will be providing additional material here soon.: https://aka.ms/OAddinPatterns
```
Comment (davidchesnut) [inline README.md]:
```suggestion
> To ensure that the community has some content and solutions for the common patterns and practices for Office Add-in, we have consolidated the content and will be providing additional material here soon.
```
Add the link later once it is operational.

**Comment by lindalu-MSFT** [comment]
@davidchesnut I approved and merged your changes. Not sure why it's still showing change requested. Can you check everything, then remove the request?

File: README.md [hunk 5]
```diff
@@ -1,21 +1,16 @@
...
 
-[![Build Status](https://travis-ci.org/OfficeDev/office-js-helpers.svg?branch=master)](https://travis-ci.org/OfficeDev/office-js-helpers)
-[![npm version](https://badge.fury.io/js/%40microsoft%2Foffice-js-helpers.svg)](https://www.npmjs.com/package/@microsoft/office-js-helpers)
-[![dependencies](https://david-dm.org/officedev/office-js-helpers.svg)](https://david-dm.org/officedev/office-js-helpers)
-[![downloads](https://img.shields.io/npm/dt/@microsoft/office-js-helpers.svg)](https://www.npmjs.com/package/@microsoft/office-js-helpers)
+> ## Can I still use this code?
+> If you find these samples, patterns, and convenience functions useful, we encourage you to take the code under your wing, improve them, and use it on your own. Even though Microsoft will no longer be supporting this content going forward, you may use the code at your convenience, external to the package.
```
Comment (davidchesnut) [inline README.md]:
```suggestion
> **Security vulnerabilities may exist in the project, or its dependencies.** If you plan to reuse or run any code from this repo, be sure to perform appropriate security checks on the code or dependencies first.
> If you find these samples, patterns, and convenience functions useful, we encourage you to take the code under your wing, improve them, and use it on your own. Even though Microsoft will no longer be supporting this content going forward, you may use the code at your convenience, external to the package.

```

**Pull request merged**